packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -15,49 +15,24 @@
     </oui-message>
 
     <oui-message
-        data-ng-if="$ctrl.users.length === 0 && $ctrl.allowedIps.length > 0"
+        data-ng-if="$ctrl.getWarningMessage()"
         data-type="warning"
         data-dismissable
     >
-        <span
-            data-translate="pci_databases_general_information_no_users"
-        ></span>
-        <a href data-ng-click="$ctrl.manageUsers()">
-            <span
-                data-translate="pci_databases_general_information_manage_users"
-            ></span>
-        </a>
-    </oui-message>
-
-    <oui-message
-        data-ng-if="$ctrl.allowedIps.length === 0 && $ctrl.users.length > 0"
-        data-type="warning"
-        data-dismissable
-    >
-        <span data-translate="pci_databases_general_information_no_ip"></span>
-    </oui-message>
-
-    <oui-message
-        data-ng-if="$ctrl.allowedIps.length === 0 && $ctrl.users.length === 0"
-        data-type="warning"
-        data-dismissable
-    >
-        <span
-            data-translate="pci_databases_general_information_no_ip_no_users"
-        ></span>
-        <a href data-ng-click="$ctrl.manageUsers()">
-            <span
-                data-translate="pci_databases_general_information_manage_users"
-            ></span>
+        <span data-translate="{{$ctrl.getWarningMessage().message}}"></span>
+        <a
+            data-ng-if="$ctrl.getWarningMessage().userLink"
+            href
```

Comment:
Please use a button if there are no href

Reply:
We already had a problem like this one before, and it seems that using a button in a oui-message necessitate custom css for proper display:
![image](https://user-images.githubusercontent.com/24541481/139886246-9d00c2b0-fe0e-4d87-bb1c-3cfd721b7b11.png)

Howerver, having `href=""` in place of `href` makes the html valid, so that is what I changed in this commit: https://github.com/ovh/manager/pull/5872/commits/a05c7a3c10e785323d656e91bcf3b6ffb007a133

Reply:
**nitpick:** This should work:
```html
<button class="oui-link" type="button"></button>
```

Reply:
Works like a charm, thanks ! https://github.com/ovh/manager/pull/5872/commits/899609c78fda9697a820ff2e6163155760d434f5